src/state.py

```diff
@@ -18,6 +18,15 @@ def __str__(self):
             print(i, student)
         return st 
 
+    def set_score(self):
+        self.score = self.get_score()
```

Comment:
is this correct? insere um comentário aqui por favor para ficar mais claro o que está a acontecer

Reply:
@LuisMMMTS This is meant to set the state score to a variable.
If you want to access the state score more than once, it's important that you do not call a costy function, like the heuristic, more than once.
For that purpose, the heuristic function is called once and stored for later use.

Reply:
Despite this, this was an error, as this function comes from a different PR which was accidentally merged here. I'm removing it for now !